bciers/libs/components/src/form/FormBase.tsx

```diff
@@ -22,6 +27,10 @@ export const customFormatsErrorMessages = {
   phone: "Format should be ### ### ####",
   email: "Please enter a valid email address, e.g. mail@example.com",
   uri: "Please enter a valid website link, e.g. http://www.website.com, https://www.website.com",
+  starting_date_format: "Starting Date format should be YYYY-MM-DD",
```

Comment:
Recommend making this just a generic `date_format` so it can be used for any fields that use the date picker.
```suggestion
  date_format: "Date format should be YYYY-MM-DD",
```

Reply:
Edited!